kolibri/core/errorreports/middleware.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import traceback
+
+from django.db import IntegrityError
+
+from .models import ErrorReports
+
+
+class ErrorReportingMiddleware:
+    """
+    Middleware to log exceptions to the database.
+    ref: https://docs.djangoproject.com/en/5.0/topics/http/middleware/#writing-your-own-middleware
```

Comment:
Noting that the Django version here should be 3.2 and not 5.0. But also, we should remove this reference as having it doesn't enhance the documentation in any way.

`ref: https://docs.djangoproject.com/en/5.0/topics/http/middleware/#writing-your-own-middleware`

Reply:
Nice catch!